noxfile.py

```diff
@@ -141,7 +139,7 @@ def docs(session):
     """Build the docs for this library."""
 
     session.install("-e", ".")
-    session.install("sphinx", "alabaster", "recommonmark")
+    session.install("sphinx<3.0.0", "alabaster", "recommonmark")
```

Comment:
Pinning Sphinx to `< 3.x` is not needed anymore, we fixed all the warning in https://github.com/googleapis/python-pubsub/pull/70.

Not a show-stopper, though, just something we can clean in the next autogen pass.